hera_sim/vis.py

```diff
@@ -162,7 +163,7 @@ def hmap_to_I(h):
 #     return gains, true_vis, data
 
 
-# XXX from hera_cal.redcal
+# from hera_cal.redcal
 def sim_red_data(reds, gains=None, shape=(10, 10), gain_scatter=0.1):
     """
     Simulate noise-free random but redundant (up to differing gains) visibilities.
```

Comment:
Are these noise-free? I see you call `noise.white_noise` in the body of the function.

Reply:
Yes. The visibilities and gains are random, but the baselines that should be redundant are perfectly redundant. I'll clarify that it's thermal noise free.